src/acuvity/models/scanresponsewrapper.py

```diff
@@ -0,0 +1,38 @@
+from acuvity.verdict_processing.core.guard_processor import CheckEvaluator
+from acuvity.verdict_processing.core.response_processor import GuardCheck
+from acuvity.verdict_processing.util.response_parser import ResponseParser
+from acuvity.verdict_processing.util.threshold_helper import ThresholdHelper
+
+from .scanresponse import Scanresponse
+
+
+class ScanResponseWrapper:
```

Comment:
I don't particularly like this class name. Can we call this something more meaningful, like `ScanResponseWithVerdict` please?